apps/site/snippets/en/download/nvm.bash

```diff
@@ -4,6 +4,9 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
 # Download and install Node.js:
 nvm install ${props.release.major}
 
+# activate nvm (in lieu of restarting the shell)
+\. "${HOME}/.nvm/nvm.sh"
```

Comment:
Is this really needed? 🤔

Is it guaranteed for this script to work? And why does nvm itself doesn't do that?

Reply:
Yes, it's really needed. nvm *can't* do that - the current shell environment can't be made to source something by an invoked program or script.

Reply:
ah actually i need to bump it up a few lines :-) basically, the install script requires you to restart the shell, or source nvm.sh, before the `nvm` command is available.